tests/ManageCourses.Tests/SmokeTests/SystemUnderTest.cs

```diff
@@ -22,14 +22,13 @@
 
 namespace GovUk.Education.ManageCourses.Tests.SmokeTests
 {
-
     [TestFixture]
-    public class DataServiceExportTests : ApiSmokeTestBase
+    [Category("Smoke")]
+    [Explicit]
+    public class SystemUnderTest : ApiSmokeTestBase
```

Comment:
I'm not keen on the `SystemUnderTest` class name. I think it's too obscure. What was wrong with the original name? I thought it was more descriptive.

Reply:
You don't really want to have to spin up a server for each test or test section. 

Consider will one endpoint vs another endpoint require a different server setup?

Reply:
Is there no way we can keep the same server while still having well factored code?

Reply:
Beyond immediate scope?

Reply:
This file in its entire is ugly.